WPThemeReview/Tests/CoreFunctionality/ThemeTemplatesException/footer_widgets.inc

```diff
@@ -0,0 +1,4 @@
+<!-- Annotation has to be on the second line as errors are thrown on line 1 and errors on annotation lines are ignored. -->
+phpcs:set WordPress.Files.FileName is_theme true
+<?php
```

Comment:
You need to add some code to trigger the sniff so it will actually be tested.
Try something like:
```php
$my_theme_var = 123; // OK, prefixed.
$var = // OK, template file.
```

Reply:
Although this should be in the `header.inc` as that is considered a template file, while `footer_widgets.inc` isn't, right?

Reply:
It should be in all the test files for this sniff, but the `OK` should be changed to `Bad` for non-template files.

If the code is not in non-template files, the sniff wouldn't be triggered and we still wouldn't be testing if the "fall through to the parent sniff if the file is not a template file" works correctly.